$NetBSD: patch-Makefile,v 1.1 2022/07/29 06:04:25 rin Exp $

- Do not hardcode paths and tools.
- Switch from imlib2-config to pkg-config.

--- Makefile.orig	2004-05-20 06:23:50.000000000 +0900
+++ Makefile	2022-07-29 02:24:37.776750501 +0900
@@ -1,10 +1,7 @@
-PREFIX = /usr/local
 BINDIR = ${PREFIX}/bin
 
-X11PREFIX = /usr/X11R6
-
-CFLAGS += -I${X11PREFIX}/include `imlib2-config --cflags` `pkg-config xft --cflags` -g -pedantic -Wall
-LDFLAGS += -L${X11PREFIX}/lib -lX11 -lm `imlib2-config --libs` `pkg-config xft --libs`
+CFLAGS += -I${X11BASE}/include `pkg-config imlib2 --cflags` `pkg-config xft --cflags` -pedantic -Wall
+LDFLAGS += -L${X11BASE}/lib -lX11 -lm `pkg-config imlib2 --libs` `pkg-config xft --libs`
 
 # Comment these out to disable Xinerama support
 CFLAGS += -DXINERAMA
@@ -21,11 +18,11 @@ HEADERS = skippy.h wm.h dlist.h mainwin.
 all: skippy${EXESUFFIX}
 
 skippy${EXESUFFIX}: Makefile ${SOURCES} ${HEADERS}
-	gcc ${CFLAGS} -o skippy${EXESUFFIX} ${SOURCES} ${LDFLAGS}
+	gcc ${CPPFLAGS} ${CFLAGS} -o skippy${EXESUFFIX} ${SOURCES} ${LDFLAGS}
 
 clean:
 	rm -f skippy${EXESUFFIX}
 
 install:
-	install -d ${DESTDIR}${BINDIR}
-	install -m 755 skippy$(EXESUFFIX) ${DESTDIR}${BINDIR}/skippy${EXESUFFIX}
+	$(BSD_INSTALL_PROGRAM_DIR) ${DESTDIR}${BINDIR}
+	$(BSD_INSTALL_PROGRAM) skippy$(EXESUFFIX) ${DESTDIR}${BINDIR}/skippy${EXESUFFIX}
